lib/foreman_discovery/engine.rb

```diff
@@ -119,13 +113,6 @@ class Engine < ::Rails::Engine
     end
 
     initializer "foreman_discovery.apipie" do
-      # this condition is here for compatibility reason to work with Foreman 1.4.x
-      # Also need to handle the reverse of the 1.8 method above
-      unless (SETTINGS[:version].to_s.include?('develop') or Gem::Version.new(SETTINGS[:version]) >= Gem::Version.new('1.8'))
-        if Apipie.configuration.api_controllers_matcher.is_a?(Array)
-          Apipie.configuration.api_controllers_matcher << "#{ForemanDiscovery::Engine.root}/app/controllers/api/v2/*.rb"
```

Comment:
Yeah I miss-read the `unless` block. Thought it was `if`. There is a new PR opened.